### PR TITLE
added packages/lz4_chans/lz4_chans.3.0.0

### PR DESCRIPTION
**Pull request by UnixJunkie**
updated dolog dependency version, also, should compile on more ocaml versions

### CODE REVIEW

File: packages/lz4_chans/lz4_chans.3.0.0/opam [hunk 1]
```diff
@@ -0,0 +1,26 @@
+opam-version: "2.0"
+authors: "Francois Berenger"
+maintainer: "unixjunkie@sdf.org"
+homepage: "https://github.com/UnixJunkie/lz4-chans"
+bug-reports: "https://github.com/UnixJunkie/lz4-chans/issues"
+dev-repo: "git+https://github.com/UnixJunkie/lz4-chans.git"
+license: "BSD-3-Clause"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  [make "test"] {with-test}
```
Comment (kit-ty-kate) [inline packages/lz4_chans/lz4_chans.3.0.0/opam]:
```suggestion
  ["dune" "exec" "-p" name "src/test.exe"] {with-test}
```
`dune build` without `-p` or `--profile=release` promotes some warnings as errors. This should be avoided in releases.

**Comment by kit-ty-kate** [comment]
There is an error with ocaml 4.03:
```

#=== ERROR while compiling lz4_chans.3.0.0 ====================================#
# context              2.0.5 | linux/x86_64 | ocaml-base-compiler.4.03.0 | pinned(https://github.com/UnixJunkie/lz4-chans/archive/v3.0.0.tar.gz)
# path                 ~/.opam/4.03/.opam-switch/build/lz4_chans.3.0.0
# command              /usr/bin/make test
# exit-code            2
# env-file             ~/.opam/log/lz4_chans-29-e7966f.env
# output-file          ~/.opam/log/lz4_chans-29-e7966f.out
### output ###
# dune build src/test.exe
# _build/default/src/test.exe
# 2019-10-03 07:29:10.676 INFO : plain_fn: /tmp/lz4_chans_test_3d0189.bin
# 2019-10-03 07:29:15.871 INFO : plain output: 1925094.56 float/s
# Fatal error: exception Failure("input_value: truncated object")
# make: *** [Makefile:22: test] Error 2
```

File: packages/lz4_chans/lz4_chans.3.0.0/opam [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+]
+depends: [
```
Comment (kit-ty-kate) [inline packages/lz4_chans/lz4_chans.3.0.0/opam]:
```suggestion
depends: [
  "ocaml"
```

**Comment by UnixJunkie** [comment]
I'll check what happens with 4.03.0.

**Comment by UnixJunkie** [comment]
Ok, this is due to a change in behavior of the Marshal module that was introduced in 4.03.0.
It works prior to this version, and after.
I'll disable the tests just for that version, if I manage to do it.

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**